src/appleseed/renderer/modeling/project/projectfilereader.cpp

```diff
@@ -3153,6 +3153,28 @@ namespace
 
         return false;
     }
+
+    // returns filename of .appleseed file inside archive
+    // returns "" in case there are more than 1 .appleseed files
+    // "" means it's not a valid packed project
+    string get_project_name(const char* project_filepath)
```

Comment:
`get_project_filename_from_archive()` maybe? `get_project_name()` is a bit too generic...